projects/plugins/jetpack/changelog/update-spotify-shortcode

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: other
+
+jetpack_spotify_shortcode: Prevent fatal error in PHP 8.1 if $atts is not an array
```

Comment:
It might make more sense to readers of the changelog like this:
```suggestion
Spotify shortcode: Prevent a fatal error in PHP 8.1 if no attributes are passed.
```